src/workflows/recipes/pages/RecipeListingPage.js

```diff
@@ -28,9 +28,11 @@ import {
   getQueryParam,
   getQueryParamAsInt,
 } from 'console/state/router/selectors';
+import { getAllActions } from 'console/state/actions/selectors';
 
 @connect(
   (state, props) => ({
+    actions: getAllActions(state, new Map()),
```

Comment:
Can/should this be `actions: getAllActions(state) || new Map()` instead?

Reply:
it could be that too.